blocks/bricks/conv.py

```diff
@@ -177,6 +180,144 @@ def num_output_channels(self):
         return self.num_filters
 
 
+class ConvolutionalTranspose(Initializable):
+    """Performs the transpose of a 2D convolution.
+
+    Parameters
+    ----------
+    filter_size : tuple
+        The height and width of the filter (also called *kernels*).
+    num_filters : int
+        Number of filters per channel.
+    num_channels : int
+        Number of input channels in the image. For the first layer this is
+        normally 1 for grayscale images and 3 for color (RGB) images. For
+        subsequent layers this is equal to the number of filters output by
+        the previous convolutional layer. The filters are pooled over the
+        channels.
```

Comment:
I'm confused as to whether this refers to the output (of the transpose operation) or the input (of the transpose operation). I assume the former. So we should be clearer.